$OpenBSD: patch-CMakeLists_txt,v 1.1 2018/12/27 20:30:55 rsadowski Exp $

Disable KDE4 dolphin, kdialog, keditbookmarks. Please see in
x11/kde-applications

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -37,11 +37,7 @@ if ( KActivities_FOUND )
 endif ( KActivities_FOUND )
 
 add_subdirectory( lib )
-add_subdirectory( dolphin )
-add_subdirectory( kdialog )
-add_subdirectory( keditbookmarks )
 add_subdirectory( konqueror )
-add_subdirectory( kfind )
 add_subdirectory( plasma )
 add_subdirectory( konq-plugins )
 
